snapcraft/plugins/kbuild.py

```diff
@@ -125,7 +125,7 @@ def __init__(self, name, options, project):
     def enable_cross_compilation(self):
         self.make_cmd.append('ARCH={}'.format(
             self.project.kernel_arch))
-        if 'CROSS_COMPILE' in os.environ:
+        if os.environ.get('CROSS_COMPILE'):
```

Comment:
That looks sensible.

Can you please add a variant of `test_override_cross_compile` to `snapcraft/tests/unit/plugins/test_kernel.py` which sets an empty value?

Reply:
Done.